apim-apk-agent/internal/utils/apis_fetcher.go

```diff
@@ -64,7 +64,7 @@ func FetchAPIsOnEvent(conf *config.Config, apiUUID *string, k8sClient client.Cli
 		GetAPI(c, nil, envs, sync.RuntimeArtifactEndpoint, true)
 	}
 	data := <-c
-	logger.LoggerMsg.Info("Receiving data for an API")
+	logger.LoggerUtils.Info("Receiving data for an API")
```

Comment:
Lets remove this or add a debug log with api ID

Reply:
fixed